public/stylesheet.css

```diff
@@ -216,11 +219,86 @@ a {
   text-decoration: none;
 }
 
+/* Modal */
+
+.popup__top {
+  top: -webkit-calc(var(--header-top-offset) + 65px);
+  top: -moz-calc(var(--header-top-offset) + 65px);
+  top: calc(var(--header-top-offset) + 65px);
+  right: var(--side-offset);
+}
+@media (max-width: 860px) {
+  .popup__top {
+    top: -webkit-calc(var(--header-top-offset) + 65px);
+    top: -moz-calc(var(--header-top-offset) + 65px);
+    top: calc(var(--header-top-offset) + 65px);
+  }
+}
+@media (max-width: 640px) {
+  .popup__top {
+    top: -webkit-calc(var(--header-top-offset) + 65px);
+    top: -moz-calc(var(--header-top-offset) + 65px);
+    top: calc(var(--header-top-offset) + 65px);
+  }
+}
+
+.popup-trigger {
+  display: inline-block;
+}
+
+.popup_modal {
+  background-color: #fff;
+  position: absolute;
+  border: 0.13rem solid var(--color-button-border);
+  z-index: 2011;
+  background-color: var(--color-background);
+  border-radius: var(--border-radius-large);
+}
+
+.popup__close {
+  font-size: 1.2rem;
+  cursor: pointer;
+  z-index: 2011;
+}
+
+.is__visible {
+  opacity: 1;
+  pointer-events: auto;
+}
+
```

Comment:
Is this really needed?

```suggestion
```

Reply:
It doesn't seem to be used. I don't know what this class is.